### PR TITLE
chore: update CODEOWNERS, remove johannes-gehrs

### PR DESCRIPTION
**Pull request by cbrgm**

### CODE REVIEW

**Review by djablonski-moia — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -7,4 +7,4 @@
 # the repo. Unless a later match takes precedence,
 # mentioned account names will be requested for
 # review when someone opens a pull request.
-*   @msiuts @bemica @snowiow @guusvw @djablonski-moia @oppermax @cbrgm @johannes-gehrs 
+*   @msiuts @bemica @snowiow @guusvw @djablonski-moia @oppermax @cbrgm
```
Comment (oppermax) [inline CODEOWNERS:10]:
``` suggestion
*   @msiuts @bemica @snowiow @guusvw @djablonski-moia @oppermax @cbrgm @karo-moia 
```

**Review by oppermax — CHANGES_REQUESTED** [review]

**Review by djablonski-moia — DISMISSED** [review]

**Comment by djablonski-moia** [comment]
The failing checks can't be related to the change is this PR :thinking:

**Comment by cbrgm** [comment]
Yeah I'll do a rebase

**Comment by djablonski-moia** [comment]
@oppermax needs to review anyway, since he requested changes :wink:

**Review by oppermax — APPROVED** [review]

**Pull request merged**